src/pages/unseoJang/Login/Login.js

```diff
@@ -2,14 +2,82 @@ import './Login.scss';
 import React, { Component } from 'react';
 
 class Login extends Component {
+  //login handle
+  constructor() {
+    super();
+    this.state = {
+      //state 말 그대로 컴포넌트의 상태를 수정
```

Comment:
불필요한 주석은 삭제하시는게 좋습니다!
학습을 위해서 주석 남겨두신건가요?

Reply:
네 나중의 학습을 위해 주석을 남겨 놓은겁니다.

```suggestion
      //state 말 그대로 컴포넌트의 상태를 수정
```